ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -524,13 +519,13 @@ objects:
               mv /tmp/artifacts/installer/openshift_install_state_updated.json /tmp/artifacts/installer/.openshift_install_state.json
           fi
 
-          if [ -f /tmp/artifacts/installer/terraform.tfstate ]
+          if [ -f /tmp/shared/tf/terraform.tfstate ]
```

Comment:
Is there a reason to diverge from [metal][1] for this directory?

[1]: https://github.com/openshift/release/pull/3587/files#diff-1c5ad26c50e4b0874d0c277f2152d68fR376

Reply:
Reworked the template to use the same path